hazelcast/src/test/java/com/hazelcast/test/HazelcastTestSupport.java

```diff
@@ -192,14 +192,24 @@ public static Config smallInstanceConfig() {
         return shrinkInstanceConfig(new Config());
     }
 
-    public static Config shrinkInstanceConfig(Config config) {
+    public static Config smallInstanceConfigWithoutJetAndMetrics() {
+        return smallInstanceConfigWithoutJetAndMetrics(new Config());
+    }
+
+    private static Config smallInstanceConfigWithoutJetAndMetrics(Config config) {
         // make the test instances consume less resources per default
         config.setProperty(ClusterProperty.PARTITION_COUNT.getName(), "11")
                 .setProperty(ClusterProperty.PARTITION_OPERATION_THREAD_COUNT.getName(), "2")
                 .setProperty(ClusterProperty.GENERIC_OPERATION_THREAD_COUNT.getName(), "2")
                 .setProperty(ClusterProperty.EVENT_THREAD_COUNT.getName(), "1");
-        config.getJetConfig().setEnabled(true).setCooperativeThreadCount(2);
+        config.getMetricsConfig().setEnabled(false);
+        config.getJetConfig().setEnabled(false);
+        return config;
+    }
 
+    public static Config shrinkInstanceConfig(Config config) {
+        smallInstanceConfigWithoutJetAndMetrics(config);
+        config.getJetConfig().setEnabled(true).setCooperativeThreadCount(2);
```

Comment:
also restore metrics to enabled here, otherwise `smallInstanceConfig()` won't have metrics enabled (as it used to have)

Reply:
done